src/templates/status.html

```diff
@@ -26,4 +29,21 @@
                 }
             })
     }
+
+    function getQueuePosition(runId) {
+        $.ajax(
+            {
+                url: "./" + runId + "/position/",
+                success: function (result) {
+                    let queuePositionString = "Your job is scheduled after " + result + " other runs";
+                    if (result == "0") {
+                        queuePositionString = "Your job is next";
+                    }
+                    $("#status-" + runId).html("<i class=\"fas fa-sync-alt\"></i> " + queuePositionString);
+                },
+                error: function () {
+                    $("#status-" + runId).html("<i class=\"fas fa-sync-alt\"></i> Your run is scheduled. Queue position unknown.");
```

Comment:
To make the code a little less cluttered, do you mind extracting out `$("#status-" + runId).html("<i class=\"fas fa-sync-alt\"></i> Button text.");` as a function used to update the button content?  It can also take another argument for the button style and update `.attr("class", ..)` as well.